pkg/model/form_submission.go

```diff
@@ -0,0 +1,81 @@
+package model
+
+import (
+	"fmt"
+	"gopkg.in/mgo.v2/bson"
+	"time"
+)
+
+// this is what we expect for input for a form submission
+type FormSubmissionAnswerInput struct {
+	WidgetId int64       `json:"widget_id"`
+	Answer   interface{} `json:"answer"`
+}
+
+type FormSubmissionInput struct {
+	FormId  string                      `jsont:"form_id"`
+	Status  string                      `json:"status" bson:"status"`
+	Answers []FormSubmissionAnswerInput `json:"replies"`
+}
+
+// here's what a form submission is
+type FormSubmissionAnswer struct {
+	WidgetId     int64       `json:"widget_id" bson:"widget_id"`
+	Answer       interface{} `json:"answer" bson:"answer"`
+	EditedAnswer interface{} `json:"edited" bson:"edited"`
```

Comment:
Answer is the original answer by the submitter.  EditedAnswer is a place to store edits made by staff/journalists.  This functionality will be built out further in the near future.